CHANGELOG.md

```diff
@@ -6,8 +6,28 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 4.4.0
+
+### General
+
+* `FEAT`: enable `Backspace` on MacOS for element removal ([#1989](https://github.com/camunda/camunda-modeler/issues/1989))
+* `FEAT`: enable `SHIFT + click` for multi-selection ([#1964](https://github.com/camunda/camunda-modeler/issues/1964))
+* `FIX`: correct switching between diagram and XML ([#1925](https://github.com/camunda/camunda-modeler/issues/1925))
+* `CHORE`: capture deployment and process variables in telemetry events
+
+### BPMN
+
+* `FIX`: correct removal of element templates from events ([#1990](https://github.com/camunda/camunda-modeler/issues/1990))
+* `FIX`: unlink incompatible element template during replace ([#1961](https://github.com/camunda/camunda-modeler/issues/1961))
 * `FIX`: store variable events for conditional events in the correct property ([#836](https://github.com/camunda/camunda-modeler/issues/836))
 * `FIX`: re-enable `entriesVisible` property of templates ([#1975](https://github.com/camunda/camunda-modeler/issues/1975))
+* `FIX`: correct label of start instance tool ([#1777](https://github.com/camunda/camunda-modeler/issues/1777))
+
+### DMN
+
+* `FEAT`: allow decision table name to take empty space
```

Comment:
Any reason we don't reference https://github.com/bpmn-io/dmn-js/issues/579 here?
(It is not in camunda-modeler, but we also did that in the past?)

Reply:
Sorry for that, will add the library version bumps. 

In the past, we did not reference these library issue. Instead we simply stated what was changed and referred to the actual library release.